app/views/observer_mailer/observer_added_notification.html.haml

```diff
@@ -0,0 +1 @@
+<!-- TODO activity template -->
```

Comment:
this is where we would use the standard "activity happened" template with some local `message`